blueprints/app/files/.eslintrc.js

```diff
@@ -3,14 +3,14 @@
 module.exports = {
   root: true,
   parser: '<%= typescript ? '@typescript-eslint/parser' : 'babel-eslint' %>',
-  parserOptions: {
+<% if (!typescript) { %>  parserOptions: {
     ecmaVersion: 2018,
```

Comment:
`@typescript-eslint/parser` also has a [`ecmaVersion` option](https://github.com/typescript-eslint/typescript-eslint/tree/main/packages/parser#parseroptionsecmaversion), but it already defaults to `2018` so I just went for no configuration options for now.

Reply:
`latest` makes sense to me. After all we allow the *authoring* standard to include anything that `@babel/preset-env` will later be able to transpile away (if needed), right? FWIW, we also have `latest` in the v2 addon blueprint.

Also, I have a slight preference to leave the `ecmaVersion` in place explicitly, even if it matches the defaults. Doesn't make a difference for ESLint, but maybe helps to communicate the intent to users?

Reply:
I've added the `ecmaVersion` option again and will make the switch to `latest` in a separate PR once this one is merged.